apps/admin/frontend/src/screens/contest_adjudication_screen.tsx

```diff
@@ -367,9 +374,9 @@ export function ContestAdjudicationScreen(): JSX.Element {
   const voteCount = Object.values(voteState).filter(Boolean).length;
   const isOvervote = voteCount > seatCount;
 
-  const writeInOptionIds = Array.from({ length: seatCount }).map(
-    (_, idx) => `write-in-${idx}`
-  );
+  const writeInOptionIds = useMemo(() => iter(allContestOptions(contest))
```

Comment:
Is the use of `useMemo` here due to an observed bug or performance issue? If it's just speculative, I'd remove it.